### DISCUSSION TITLE
Customfields generate column name in camelCase instead of Capital case

### DISCUSSION THREAD

**Discussion by minuth** [vendurehq/vendure#2799] [Q&A]
- Database: Postgres

- Custom fields config:
```
{
        name: "sellerType",
        type: "string",
 }
```
- Generated column name in migration file: `customFieldsSellertype`

- Expected result: `customFieldsSellerType`

**Comment by michaelbromley** [comment]
Hi,

This is an implementation detail of TypeORM, and not something that Vendure has direct control over.

**Reply by minuth** [reply]
Thanks @michaelbromley  for the reply, But I wonder why the migration generated from the default Vendure entities is correct.

**Reply by michaelbromley** [reply]
what do you mean by that?

**Reply by minuth** [reply]
I mean this issue only happened with custom field via Vendure configuration
